data/human/ships.txt

```diff
@@ -1260,6 +1260,61 @@ ship "Cruiser"
 
 
 
+ship "Cutthroat"
+	sprite "ship/pirate cutthroat"
+	thumbnail "thumbnail/pirate cutthroat"
+	attributes
+		category "Light Warship"
+		"cost" 1100000
+		"shields" 2800
+		"hull" 600
+		"required crew" 4
+		"bunks" 6
+		"mass" 171
+		"drag" 3
```

Comment:
```suggestion
		"drag" 3.8
```
Considering the ship is *gaining* outfit, engine, and mass (even visibly), I don't think the drag should be *lower*, but a tiny bit higher. This shouldn't be a problem considering its new engine and afterburner outfit.

Reply:
Eh, depending on what drag actually is. Is it just built-in speed limiter for the ship or is it a law of the universe. Also Marauders have zero drag change despite all the ships being heavier *also* a big clunky and "slow" Marauder Engine Leviathan literally outruns interceptor and light warships somehow, it is utterly ridiculous that its top speed jumps from 350 to 1000. The Cutthroat is supposed to be one of the fastest ship around and I think it definitely shouldn't be caught up so easily by such a big ship. (Even right now default variant is still slower)

Reply:
Fair enough.

Reply:
I think there's a strong case to be made that all (or at least most) of the marauder ships need to have their drag boosted a bit.